src/main/java/net/openhft/chronicle/queue/reader/ChronicleReader.java

```diff
@@ -492,15 +516,16 @@ private ChronicleQueue createQueue() {
                 .build();
     }
 
-    protected void applyFiltersAndLog(final String text, final long index, AtomicLong matches) {
+    private boolean applyFiltersAndLog(final long index, final String text) {
```

Comment:
I changed this API so the MessageConsumer is no longer responsible for updating the match counter, instead a boolean is returned to indicate whether something made it to the sink or not. The caller keeps track of how many made it.

Reply:
Much simpler

Reply:
Kinda changed it back, but in a nicer way